src/pages/api/feedback/index.page.ts

```diff
@@ -1,15 +1,44 @@
 import { NextApiRequest, NextApiResponse } from 'next';
+import prisma from '../../../api/prisma/prisma';
 
 async function handler(req: NextApiRequest, res: NextApiResponse) {
-  const feedback = req.body;
-  fetch(process.env.SLACK_FEEDBACK_WEBHOOK, {
-    method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
-    body: JSON.stringify({ text: feedback?.text }),
-  });
-  res.send(200);
+  const { text, email, route } = req.body;
+
+  if (typeof text !== 'string' || text.length === 0) {
+    return res.status(400).send('Feedback text is required');
```

Comment:
as we're adding the check, we might want to bubble this up to our form on the frontend

Reply:
I'll add a frontend check for text in the textarea before a submission is allowed to be submitted, so we have this backend check as well as a frontend message

Reply:
here's the check:

https://github.com/public-convenience-ltd/toiletmap/pull/1695/files#diff-c490ab266c5906e882ea09e434550470651318e8a7c87b02823b61ecb1fbcf18R63-R65

Reply:
Probably not blocking though, would be happy if we span out into a separate issue :)

Reply:
sounds good, I'll make an issue to capture 🆒